src/utils.js

```diff
@@ -1326,5 +1326,28 @@ export function cyrb53Hash(str, seed = 0) {
 export function isAllowZeroCpmBidsEnabled(bidderCode) {
   const bidderSettings = getGlobal().bidderSettings;
   return ((bidderSettings[bidderCode] && bidderSettings[bidderCode].allowZeroCpmBids === true) ||
-   (bidderSettings.standard && bidderSettings.standard.allowZeroCpmBids === true));
+    (bidderSettings.standard && bidderSettings.standard.allowZeroCpmBids === true));
+}
+
+/**
+ * Return a memoized version of the given function that uses the first argument as cache key.
+ */
+export function memoize1(fun) {
+  const cache = {};
+  return function (a0, ...rest) {
+    if (!cache.hasOwnProperty(a0)) {
+      cache[a0] = fun(a0, ...rest);
+    }
+    return cache[a0];
+  }
+}
+
+/**
+ * Return a memoized version of the given zero-arg function.
+ */
+export function cache(fun) {
```

Comment:
Do we need both cache and memoize?  
It seems to me that just `memoize` function is enough?